subprojects/launcher/src/integTest/groovy/org/gradle/launcher/continuous/jdk7/SymlinkContinuousIntegrationTest.groovy

```diff
@@ -74,11 +81,19 @@ class SymlinkContinuousIntegrationTest extends AbstractContinuousIntegrationTest
         def targetDir = baseDir.file("target").createDir()
         targetDir.files("A", "B")*.createFile()
 
-        def symlink = baseDir.createDir("linkdir").file("link")
+        def linkdir = baseDir.createDir("linkdir")
+        def symlink = linkdir.file("link")
+        // Since we remove symlinks at the end of the build from the VFS, we
+        // need an existing sibling of the symlink to ensure the parent directory of
+        // the symlink is watched between builds.
+        linkdir.file("existing").createFile()
```

Comment:
Essentially, if there are only symlinks in a directory, we don't watch it, right? I can imagine this leading to some surprises. I wonder how we could document this the best. Feels like we should have a list of "special cases" or something in the continuous build documentation perhaps.